modules/local/collapseHV/main.nf

```diff
@@ -5,6 +5,8 @@ process COLLAPSE_HV {
     memory "16.GB"
     input:
         path(hv_hits_filtered)
+        path(merged_bbmerge_paired_results)
```

Comment:
This module is really only solving one specific problem, namely the presence of discordant read pairs in the HV TSV. I'd suggest moving the new parts to a different module.

Reply:
@willbradshaw done.